### PR TITLE
Tachyons

### PR DESCRIPTION
**Pull request by aaadryyy**

### CODE REVIEW

**Comment by aaadryyy** [comment]
From the doc : https://github.com/tachyons-css/tachyons-and-react/tree/master/getting-started

File: client/src/App.js [hunk 1]
```diff
@@ -1,28 +1,12 @@
...
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
-      </div>
-    );
-  }
+	render() {
```
Comment (adipasquale) [inline client/src/App.js]: please use spaces (2) and not tabs :) 

image

File: client/package.json [hunk 2]
```diff
@@ -5,13 +5,18 @@
...
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject",
+    "css": "tachyons src/css/app.css > src/index.css --minify",
+    "prestart": "npm run css",
```
Comment (adipasquale) [inline client/package.json]: Does it work for you ? I get this weird error 'unknown browsers dead' :

screen shot 2018-12-28 at 11 28 40

It seems related with using a deprecated version of browserslist.

To be completely transparent, I'm not really sure we need to use the `tachyons-cli` package at all. can you just try to remove it and remove this `css` script thing ?

**Review by adipasquale — CHANGES_REQUESTED** [review]
The doc that you found on github is 2 years old, I have a feeling it's outdated

File: client/package.json (hunk 2 above)
Comment (aaadryyy) [reply client/package.json]: According to this issue https://github.com/tachyons-css/cli/issues/22 I had to delete this part then it worked.
`
"browserslist": [
  ">0.2%",
  "not dead",
  "not ie <= 11",
  "not op_mini all"
]
`

**Comment by aaadryyy** [comment]
Oh, true it's old.
Actually looks like all the React + Tachyons doc is old (https://github.com/tachyons-css/tachyons-and-react/tree/master/getting-started)
How would you like tachyons to be added to the project?

File: client/package.json (hunk 2 above)
Comment (aaadryyy) [reply client/package.json]: Deleted tachyons-cli + script CSS, works fine 😃

**Comment by aaadryyy** [comment]
Seems to work fine without tachyons-cli and the CSS Script 👌

**Review by adipasquale — APPROVED** [review]

**Pull request merged**